docker/conda/environments/cuda11.8_examples.yml

```diff
@@ -24,15 +24,15 @@ channels:
     - nvidia
     - conda-forge
 dependencies:
-    - boto3=1.24.59
+    - boto3=1.26.76
```

Comment:
We dont directly depend on `boto3` so we should just remove this from the environment and get whatever version our dependencies want.

Reply:
It's used directly in:
```
examples/digital_fingerprinting/production/morpheus/benchmarks/test_bench_e2e_dfp_pipeline.py
```